lib/cws/client.ts

```diff
@@ -68,6 +68,11 @@ export class WebSocket extends eventEmitter() {
       remoteFamily: address[2]
     };
   }
+  
```

Comment:
I am getting `[tslint] trailing whitespace` seems like you have them on line 71.